lua/neogit/status.lua

```diff
@@ -573,25 +573,40 @@ end
 local stage = function()
   M.current_operation = "stage"
   local section, item = get_current_section_item()
+  local mode = vim.api.nvim_get_mode()
 
   if section == nil
     or (section.name ~= "unstaged" and section.name ~= "untracked" and section.name ~= "unmerged")
-    or item == nil then
+    or (mode.mode == "V" and item == nil) then
     return
   end
 
-  local mode = vim.api.nvim_get_mode()
-
   if mode.mode == "V" then
     stage_selection()
   else
     local on_hunk = current_line_is_hunk()
-    if on_hunk and section.name ~= "untracked" then
-      local hunk = get_current_hunk_of_item(item)
-      local patch = generate_patch_from_selection(item, hunk)
-      cli.apply.cached.with_patch(patch).call()
+    if item == nil then
+      if section.name == "unstaged" then
+        git.status.stage_modified()
+--    elseif section.name == "untracked" then
```

Comment:
Here you have to through each item in the untracked section and call `git.status.stage(item.name)` on it.

Reply:
Ok so there's no way to bulk stage? I'm asking because what magit does is call only once git stage with all the untracked files.

Reply:
I guess you could build your own git command using our [CLI Api](https://github.com/TimUntersberger/neogit/blob/master/lua/neogit/lib/git/cli.lua).

Example: ```cli.add.files(...).call()``` ( or something like that )

Reply:
I managed to figure it out, sorry for wasting your time but I'm a lua noob especially with all the metatable magic that the CLI uses.